cove/locale/es/LC_MESSAGES/django.po

```diff
@@ -111,15 +112,25 @@ msgstr ""
 "\n"
 "Sólo podemos procesar archivos json, csv y xlsx."
 
-#: cove/lib/exceptions.py:53
+#: cove/lib/exceptions.py:35
 msgid ""
-"We think you tried to supply a spreadsheet, but we failed to convert it to JSON.\n"
+"We did not recognise the file type.\n"
+"\n"
+"We can only process xml, csv and xlsx files."
+msgstr ""
+"No se reconoce el tipo de archivo. \n"
+"\n"
+"Sólo podemos procesar archivos xml, csv y xlsx."
+
+#: cove/lib/exceptions.py:62
+msgid ""
+"We think you tried to supply a spreadsheet, but we failed to convert it.\n"
 "\n"
 "Error message: {}"
 msgstr ""
-"Creemos que ha tratado de suministrar una hoja de cálculo, pero no hemos podido convertirla a JSON.\n"
+"Creemos que se trató de suministrar una hoja de cálculo, pero no hemos podido convertirlo.\n"
```

Comment:
_pero no hemos podido **convertirlo**_ 
should be 
_pero no hemos podido convertirl**a**_